contracts/ccip/ccip/sources/fee_quoter.move

```diff
@@ -498,16 +510,20 @@ public fun apply_token_transfer_fee_config_updates(
 }
 
 // this should only be called from offramp, hence gated by a fee quoter cap stored in offramp
-public fun update_prices(
+public(package) fun update_prices(
     ref: &mut CCIPObjectRef,
-    _: &FeeQuoterCap,
+    // _: &FeeQuoterCap,
```

Comment:
this cap can be removed.
bc we can use `package` modifier to restrict permission control bc previously they are different packages.